libs/ml/tests/ml/serializers/ml_types.cpp

```diff
@@ -119,14 +118,12 @@ TYPED_TEST(SerializersTestNoInt, serialize_graph_saveable_params)
   TensorType tmp_prediction = g->ForwardPropagate(output);
 
   fetch::ml::GraphSaveableParams<TypeParam> gsp1 = g->GetGraphSaveableParams();
-  fetch::serializers::MsgPackSerializer     b;
-  b << gsp1;
-  b.seek(0);
+  fetch::serializers::BigSerializer b;
+  b.Serialiaze(gsp1);
```

Comment:
Lets make this wrapper have the same interface as the MsgPackSerializer i.e. streaming operators and and accessors. Then the type can simple be changed